mainchain-follower/db-sync-follower/src/candidates/datum/mod.rs

```diff
@@ -9,3 +8,13 @@ mod registered;
 
 type Error = Box<dyn std::error::Error + Send + Sync>;
 type Result<T> = std::result::Result<T, Error>;
+
+trait PlutusDataExtensions {
+	fn as_u16(self) -> Option<u16>;
+}
+
+impl PlutusDataExtensions for cardano_serialization_lib::PlutusData {
+	fn as_u16(self) -> Option<u16> {
+		u16::try_from(u32::try_from(self.as_integer()?.as_u64()?).ok()?).ok()
```

Comment:
There is TryFrom<u64> for u16. We don't need u32 in the middle.

Reply:
ther's not, sadly